### ISSUE TITLE
Issue with final burn alpha and arcade emulators vertical games

### ISSUE THREAD

**Issue by Frenky81** [jimgraygit/Onion#157]
hi everyone, I noticed that after updating to onion os 3.6 and later also to onion 3.8.2 the arcade emulators like fba or fbneo are no longer able to put games that are natively vertical in landscape mode.I think it's a bug of the retroarch porting installed in the latest versions of onion os. Has anyone noticed this thing? Could you give me explanations about it? thanks!

**Comment by dwmccoy** [comment]
Duplicate of #89 

This appears to be an issue in the FBALPHA/FBNEO core emulator. Looks like an SDLv1 video driver library problem.  This does not appear to be an issue with RetroArch.

**Comment by Frenky81** [comment]
Ok thanks for information ! I hope developers can fix this issue soon.

**Comment by Sichroteph** [comment]
Mame2003+ is a better core and can rotate automatically. If you happen to have compatibility issues, there exists romsets specifically for it.

**Comment by Frenky81** [comment]
if you use the same core both on the original software and on the onion os, same version, on the original software the vertical games are placed horizontally, on the onion no, I don't think it is a core problem, at this point it is a software problem or a retroarch issue. That being said, that it might not be as I say, I don't know, I'm not an expert but just a simple retrogaming lover.

**Comment by Wapp92** [comment]
I agree with [Sichroteph](https://github.com/Sichroteph) I tested all the cave game and they run better on the default Mame2003+.
I only wish that i could change the resolution manually, because keep aspect ratio make some vertical game really really narrow. For those games a per game resolution would be perfect.
